src/main/groovy/life/qbic/samplestatus/reporter/services/QbicSampleTrackingService.groovy

```diff
@@ -30,172 +30,175 @@ import static java.time.ZoneOffset.UTC
 @ConfigurationProperties
 class QbicSampleTrackingService implements SampleTrackingService {
 
-  @Value('${service.sampletracking.url}')
-  private String sampleTrackingBaseUrl
-
-  @Value('${service.sampletracking.location.endpoint}')
-  private String locationEndpoint
-
-  @Value('${service.sampletracking.auth.user}')
-  private String serviceUser
-
-  @Value('${service.sampletracking.auth.password}')
-  private String servicePassword
-
-  private String locationEndpointPath
-
-  @PostConstruct
-  void initService() {
-    locationEndpointPath = sampleTrackingBaseUrl + locationEndpoint
-  }
-
-  @Override
-  Optional<Location> getLocationForUser(String userId) {
-    URI requestURI = createUserLocationURI(userId)
-    HttpResponse response = requestLocation(requestURI)
-    return Optional.of(response.body()).flatMap(DtoMapper::parseLocationOfJson)
-  }
-
-  @Override
-  void updateSampleLocation(String sampleCode, Location location, String status, Instant timestamp, Person responsiblePerson) throws SampleUpdateException {
-    String locationJson = DtoMapper.createJsonFromLocationWithStatus(location, status, responsiblePerson, timestamp)
-    HttpResponse<String> response = requestSampleUpdate(createSampleUpdateURI(sampleCode), locationJson)
-    if (response.statusCode() != 200) {
-      throw new SampleUpdateException("Could not update $sampleCode to ${location.getLabel()} - ${response.statusCode()} : ${response.headers()}: ${response.body()}")
-    }
-  }
```

Comment:
why was this removed?

Reply:
This was a duplicate due to the merge, the correct method does the same without responsible person object (it's taken from the location itself)